### ISSUE TITLE
TensorFlow 2.0.0-beta0 CPU, Python 3.5, ARMv7 for Raspberry Pi

### ISSUE THREAD

**Issue by leigh-johnson** [yaroslavvb/tensorflow-community-wheels#114]
Hi y'all. Just to note, this wheel is built from v2.0.0-beta0 with an additional patch. Full details here: https://github.com/tensorflow/tensorflow/issues/29819#issuecomment-502492512

Wheel: https://github.com/leigh-johnson/tensorflow-community-wheels/raw/master/tensorflow-2.0.0b0-cp35-none-linux_armv7l.whl

https://github.com/leigh-johnson/tensorflow-community-wheels/raw/master/tensorflow-2.0.0b0-cp35-none-linux_armv7l.tar.gz

Build command:
```
git checkout v2.0.0-beta0 && git cherry-pick c6a9456b5d766adb058d81b52f832acd7397c735
CI_DOCKER_EXTRA_PARAMS="-e CI_BUILD_PYTHON=python3 -e CROSSTOOL_PYTHON_INCLUDE_PATH=/usr/include/python3.5" \
    tensorflow/tools/ci_build/ci_build.sh PI-PYTHON3 \
    tensorflow/tools/ci_build/pi/build_raspberry_pi.sh 
```

**Comment by aakbar5** [comment]
I have done cross compilation for a couple of ARM based platforms. Is it worth to send patches to include their build files? 
P.S: Build files are nicely written so one can easily extend to accommodate different toolchain.

**Comment by leigh-johnson** [comment]
Hi @aakbar5! I'm not part of the official Tensorflow team, so I'm not sure. It might make sense to open an issue in the Tensorflow repo, and describe your build targets and build file modifications.

**Comment by rcpilotp51** [comment]
Leigh, I would like to obtain a trained model of hummingbirds and take a still image and video when detected - do you know where i can find some resources to accomplish this?

**Comment by leigh-johnson** [comment]
Hey @rcpilotp51,

I don't know of any pre-trained hummingbird models, but that sounds like a nifty project. Here's how I would get started...

1. Collect a few thousand images of hummingbirds. Web and youtube scraping are a good source of raw data. 
2. Label your data with bounding boxes. You can pay a virtual assistant to do this for you, or do it yourself if your time budget is higher than your dollar budget. I wrote a guide to automating bounding box labeling with a "guidance" AutoML model. https://www.bitsy.ai/automate-bounding-box-annotation-with-tensorflow-and-automl/
3. Train an object detection model. I've found SSD + MobileNet network architechtures to be a good trade-off of performance and accuracy for offline inference on the Raspberry Pi. You can also skip this step and deploy the AutoML model directly. =)
